runtime/reference/cli/compiler.md

```diff
@@ -84,6 +84,29 @@ To include non-statically analyzable dynamic imports, specify an
 deno compile --include calc.ts --include better_calc.ts main.ts
 ```
 
+## Including Data Files or Directories
```

Comment:
```suggestion
## Including Data Files or Directories (Deno 2.1+)
```

Reply:
This should say the version. Not sure the best place to put this (just on my phone)